.github/workflows/pull_request_review.yml

```diff
@@ -33,14 +42,22 @@ jobs:
       with:
         node-version: ${{ matrix.node }}
 
+    - name: npm cache
+      uses: actions/cache@v2
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
```

Comment:
Is this missing the hash for restore ?

Reply:
https://github.com/actions/cache/blob/main/examples.md#macos-and-ubuntu